docs/integrations/persisting-store-data.md

```diff
@@ -630,6 +630,37 @@ export const useBoundStore = create(
 )
 ```
 
+If you're using a type that JSON.stringify() doesn't support, you'll need to write your own serialization/deserialization code. However, if this is tedious, you can use third-party libraries to serialize and deserialize different types of data.
+
+For example, [Superjson](https://github.com/blitz-js/superjson) can serialize data along with its type, allowing the data to be parsed back to its original type upon deserialization
+
+```ts
+import superjson from "superjson"; //  can use anything: serialize-javascript, devalue, etc.
+import { StorageValue } from "zustand/middleware";
+
+interface BearState {
+  ...: Map<string, string>;
```

Comment:
Instead of an ellipsis, use some example names, like "bears", "fish", "time", "query", or something like that.